core/domain/email_manager.py

```diff
@@ -465,3 +465,43 @@ def send_suggestion_email(
                 recipient_id, feconf.SYSTEM_COMMITTER_ID,
                 feconf.EMAIL_INTENT_SUGGESTION_NOTIFICATION,
                 email_subject, email_body, feconf.NOREPLY_EMAIL_ADDRESS)
+
+
+def send_instant_feedback_message_email(
+        recipient_id, sender_id, message, exploration_title, exploration_id,
+        thread_title):
+    email_subject_template = 'New Oppia message in "%s"'
```

Comment:
Then perhaps we might have entire different function for generating status change email body ?